src/antsibull_docs/data/docsite/plugin.rst.j2

```diff
@@ -257,21 +263,25 @@ This describes positional parameters of the test. These are the values ``positio
 .. Options
 
 {% if doc['options'] | remove_options_from_list(options_to_skip) -%}
+{%   set has_keyword_parameters = true -%}
 
-{%   if plugin_type in ['filter', 'test'] %}
+{%   if plugin_type in ['lookup', 'filter', 'test'] %}
 Keyword parameters
 ------------------
+
+This describes keyword parameters of the @{plugin_type}@. These are the values ``key1=value1``, ``key2=value2`` and so on in the following example:@{" "}@
```

Comment:
Same here, don't make this unnecessary complicated with Jinja2 whitespace handling.